core/EE_System.core.php

```diff
@@ -1199,18 +1199,10 @@ public function initialize_last()
         );
         $rewrite_rules->flushRewriteRules();
         add_action('admin_bar_init', array($this, 'addEspressoToolbar'));
-        if ($this->request->isAdmin()) {
-            $this->loader->getShared('EventEspresso\core\services\privacy\policy\PrivacyPolicyManager');
+        if($this->request->isAjax() && $this->maintenance_mode->models_can_query()) {
```

Comment:
FYI I left these methods in here because it turns out `EE_Admin` isn't loaded on admin ajax that doesn't have `ee_admin_ajax`

Reply:
just need a space after the `if` in this line in order to pass the code sniffer